docs/index.md

```diff
@@ -15,46 +15,50 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
-| visit.provider | Text     |                                                            | Full provider name, last name, or just NPI                                                                                                                                                                                  |
-| Lab Array      |          |                                                            |                                                                                                                                                                                                                             |
-| lab.lab_id     | Numeric  | 16 Digit GE ID                                             | Unique GE ID for specific lab result (ObsID)                                                                                                                                                                                |
-| lab.type       | Text     | BLL                                                        | Static BLL unless we identify additional labs to include                                                                                                                                                                    |
-| lab.date       | Date     | yyyy-mm-dd                                                 |                                                                                                                                                                                                                             |
-| lab.route      | Text     | V                                                          | *Always "V"  Veinous for Alliance sites                                                                                                                                                                                     |
-| lab.result     | Text     |                                                            | Several values are possible; integer, non-integer numeric, ranges indicated by alphanumeric text.  We'll use previous examples to essentially construct a dictionary.                                                       |
+| Field           | Format   | Constraints                        | Notes                                                                                                             |
+|-----------------|----------|------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| timestamp       | TimeDate | yyyy-mm-dd hh:mm:ss.sss-hh:mm      | Current timestamp (RFC 3339 complient date)                                                                       |
+| visit_date      | Date     | yyyy-mm-dd                         | Effective date of score                                                                                |
```

Comment:
The reason I originally suggested it was because of both past and future visit dates. But:

    past visit dates will, I believe, only be used internally for validation and that will probably happen by calling predict.py directly, no reason to do it over the API. if an external provider submits a past visit date, it seems like we'd want to handle that as an error.

    future visit dates are plausible for providers, buta future visit date won't affect the returned prediction because we don't have future models trained on future data to use for the prediction, just the current one.

So not having a visit date makes things simpler. The only reason I can see for having one in the public API is to collect data on when patients are visiting, but we have that in the visit array. Sorry to go back on my original suggestion.

Reply:
Ok - we'll omit it.